blogposts/2021/feature-removal-interactive-search-mode.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Interactive search mode is no longer supported
+description: As part of our 3.24 release, interactive search mode is no longer supported.
+author: Sourcegraph
+authorUrl: https://about.sourcegraph.com
+publishDate: 2021-01-20T10:00-07:00
+tags: [blog]
+slug: the-blog-slug
+heroImage: /blog/thumbnail-image.jpg
+published: true
+---
+
+As part of our 3.24 release, interactive search mode is no longer supported. We deprecated the feature last September, and as of January 20, 2021, it has been removed for all users. 
+
+We made this change based on feedback we have received from our customers: interactive search mode is no longer the default search mode that best serves our users. Our newer plain text search input is smarter, with improved [search query syntax](https://docs.sourcegraph.com/code_search/reference/queries), [search bar highlighting](https://vimeo.com/392761379), and [autocomplete](https://vimeo.com/374329715). Fully removing interactive search mode and making plain text search the default for all users allows us to deliver a better experience and continue improving our search interface. 
```

Comment:
@Joelkw is this the right way to  make links in markdown?

Reply:
Yep! Perfect.

Reply:
(Once you're set up with git, you can preview the local site with [these instructions](https://github.com/sourcegraph/about#developing-locally) to also test.)